src/components/GetInvolvedCard.tsx

```diff
@@ -9,23 +9,24 @@ interface Props {
 }
 
 const GetInvolvedCard = ({ data }: Props) => {
-  const { title, icon, link } = data;
+  const { title, icon, link, description } = data;
 
   const imageEl =
     icon?.fluid != null ? <Img fluid={icon.fluid} imgStyle={{ objectFit: 'contain' }} /> : null;
 
   return (
     <Link to={link ?? '#'}>
-      <Box w="auto" my={10}>
-        <Flex direction="column" align="center">
-          <Box w={312} h={246}>
-            {imageEl}
-          </Box>
-          <Text color="charcoal" textTransform="uppercase" textStyle="heading2">
-            {title}
-          </Text>
-        </Flex>
-      </Box>
+      <Flex direction="column" align="center" w={{ base: '30vw', md: 'auto' }} my="auto">
```

Comment:
Not sure what the cause is, but the images seem misaligned:
![image](https://user-images.githubusercontent.com/8410924/114498992-35fc0880-9bf3-11eb-9ad8-793edd7e713f.png)